src/com/palmergames/bukkit/towny/db/TownySQLSource.java

```diff
@@ -795,195 +796,204 @@ public boolean loadTown(Town town) {
-                if (line != null)
-                    try {
-                        town.setTag(line);
-                    } catch (TownyException e) {
-                        town.setTag("");
-                    }
-                town.setPermissions(rs.getString("protectionStatus").replaceAll("#", ","));
-                town.setBonusBlocks(rs.getInt("bonus"));
-                town.setTaxPercentage(rs.getBoolean("taxpercent"));
-                town.setTaxes(rs.getFloat("taxes"));
-                town.setHasUpkeep(rs.getBoolean("hasUpkeep"));
-                town.setPlotPrice(rs.getFloat("plotPrice"));
-                town.setPlotTax(rs.getFloat("plotTax"));
-                town.setEmbassyPlotPrice(rs.getFloat("embassyPlotPrice"));
-                town.setEmbassyPlotTax(rs.getFloat("embassyPlotTax"));
-                town.setCommercialPlotPrice(rs.getFloat("commercialPlotPrice"));
-                town.setCommercialPlotTax(rs.getFloat("commercialPlotTax"));
-                town.setSpawnCost(rs.getFloat("spawnCost"));
-                town.setOpen(rs.getBoolean("open"));
-                town.setPublic(rs.getBoolean("public"));
-                town.setAdminDisabledPVP(rs.getBoolean("admindisabledpvp"));
-                town.setAdminEnabledPVP(rs.getBoolean("adminenabledpvp"));
-
-                town.setPurchasedBlocks(rs.getInt("purchased"));
-
-                line = rs.getString("homeBlock");
-                if (line != null) {
-                    search = (line.contains("#")) ? "#" : ",";
-                    tokens = line.split(search);
-                    if (tokens.length == 3)
-                        try {
-                            TownyWorld world = getWorld(tokens[0]);
-
-                            try {
-                                int x = Integer.parseInt(tokens[1]);
-                                int z = Integer.parseInt(tokens[2]);
-                                TownBlock homeBlock = world.getTownBlock(x, z);
-                                town.forceSetHomeBlock(homeBlock);
-                            } catch (NumberFormatException e) {
-                                TownyMessaging.sendErrorMsg("[Warning] " + town.getName() + " homeBlock tried to load invalid location.");
-                            } catch (NotRegisteredException e) {
-                                TownyMessaging.sendErrorMsg("[Warning] " + town.getName() + " homeBlock tried to load invalid TownBlock.");
-                            } catch (TownyException e) {
-                                TownyMessaging.sendErrorMsg("[Warning] " + town.getName() + " does not have a home block.");
-                            }
-
-                        } catch (NotRegisteredException e) {
-                            TownyMessaging.sendErrorMsg("[Warning] " + town.getName() + " homeBlock tried to load invalid world.");
-                        }
-                }
+				line = rs.getString("residents");
```

Comment:
The spacing got a bit wonked here, if you can get it back to what it was so it doesn't doesn't affect the git blame information that would be better.